tilt.dockerfile

```
@@ -1,4 +1,4 @@
-FROM alpine@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
+FROM alpine@sha256:beefdbd8a1da6d2915566fde36db9db0b524eb737fc57cd1367effd16dc0d06d
```

I don't like to have the obscure shasum here. Given this is a development environment, I think it's fine from a security POV to use a human tag:

```suggestion
FROM alpine:3.20.3
```